tests/platform_tests/test_reload_config.py

```diff
@@ -158,13 +158,7 @@ def test_reload_configuration_checks(duthosts, enum_rand_one_per_hwsku_hostname,
     if not config_force_option_supported(duthost):
         return
 
-    timeout = None
-    if duthost.get_facts().get("modular_chassis"):
```

Comment:
@auspham @yejianquan This has been removed in this PR, as now these values can be specified per dut in the inventory files.

Reply:
Hi @liamkearney-msft , I'm concerning about this logic, previously the intention is the specified wait/timeout has the highest priority, of the wait is set as 5 here and finally it would be 5.
But after your change, the plt_control has the highest priority.

Reply:
the previous logic would always give precedence to plt_reboot_ctrl, unless `plt_reboot_ctrl_overwrite` was cleared (which it was, only in this testcase). This PR also changes the logic slightly to remove that setting in favor of something more relevant for this test, which is to bail out of the reboot command once SSH comes back up. 
I think if we have specifically set an override in the inventory file for a particular sku/test, it makes sense for it to be the highest priority.

Reply:
following discussion: Ill add an assert at the top of the function to forbid using `save_reboot = true` and `wait_for_process = false` together